pom.xml

```diff
@@ -311,12 +313,6 @@
 			</plugin>
 
 			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
-				<artifactId>findbugs-maven-plugin</artifactId>
-				<version>${findbugs.plugin.version}</version>
-			</plugin>
-
-			<plugin>
```

Comment:
I removed this as it was a duplicate of https://github.com/SERG-Delft/jpacman-framework/pull/81/files#diff-600376dffeb79835ede4a0b285078036L333